linux/org.hydrogenmusic.hydrogen.metainfo.xml.in

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.hydrogenmusic.hydrogen</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  
+  <name>Hydrogen</name>
+  <summary>The advanced drum machine</summary>
+  
+  <icon type="remote" width="32" height="32">http://hydrogen-music.org/images/favicon.png</icon>
+  <icon type="remote" width="48" height="48">http://hydrogen-music.org/images/icon48.png</icon>
+  <icon type="remote" width="64" height="64">http://hydrogen-music.org/images/icon64.png</icon>
+  <icon type="remote" width="256" height="256">http://hydrogen-music.org/images/icon256.png</icon>
+  
+  <project_license>GPL-2.0-or-later</project_license>
+  
+  <description>
+	<p>
+	  Hydrogen is both a drum sequencer for pattern-based programming
+	  and a drum synth. Its simple yet feature-rich design makes it a
+	  versatile companion, be it for quick practice at home,
+	  professional usage as fully-fledged drum machine in your studio,
+	  or drum looper on steriods for your live performance.
+	</p>
+    <p>
+      Features:
+    </p>
+	<ul>
+	  <li>First-class MIDI support</li>
+      <li>Highly customizable</li>
+      <li>Stable and performant</li>
+      <li>Loads of drumkits to choose from</li>
+      <li>In-depth control of instrument, sample, and note properties</li>
+      <li>High quality time-stretching and pitch shifting</li>
+      <li>Automate it using MIDI or OSC API</li>
+	</ul>
+  </description>
+
+  <categories>
+	<category>Audio</category>
+	<category>Midi</category>
+	<category>Sequencer</category>
+  </categories>
+
+  <keywords>
+	<keyword>drum</keyword>
+	<keyword>jack</keyword>
+  </keywords>
+
+  <url type="homepage">http://hydrogen-music.org</url>
+  <url type="bugtracker">https://github.com/hydrogen-music/hydrogen/issues/</url>
+  <url type="faq">http://hydrogen-music.org/faq/</url>
+  <url type="help">http://hydrogen-music.org/documentation/manual/manual_en.html</url>
+  <url type="translate">https://github.com/hydrogen-music/hydrogen/wiki/Translator-instructions/</url>
+  <url type="contact">http://lists.sourceforge.net/mailman/listinfo/hydrogen-devel/</url>
+
+  <releases type="external" url="https://github.com/hydrogen-music/hydrogen/blob/master/linux/org.hydrogenmusic.hydrogen.releases.xml"/>
+  
+  <supports>
+    <control>pointing</control>
+    <control>keyboard</control>
+    <control>touch</control>
+    <control>tablet</control>
+  </supports>
+
+  <content_rating type="oars-1.0"></content_rating>
+  
+  <update_contact>hydrogen-devel@lists.sourceforge.net</update_contact>
+  
+  <launchable type="desktop-id">org.hydrogenmusic.hydrogen.desktop</launchable>
```

Comment:
the name of the destkop in the repository is `org.hydrogenmusic.Hydrogen.desktop`, so this will be a problem.

Reply:
Do you know whether it is save to give the desktop file a compliant lower-case name?

It's quite unfortunate Hydrogen got a mismatching ID in the first place, as the reverse URL [should](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#tag-id-generic) be `org.hydrogen_music.hydrogen`. But I have no experience with toolchains relying on AppStream metadata and don't want to break existing workflows either.